tutorials/rise_mnist.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    pred_onnx = run_model(X_test)
</pre><p>This call errors as X_test does not have the correct dimensions for 'run_model' I think. The original X_test, before overwriting does have the correct dimensions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>